### ISSUE TITLE
Figure out what needs to happen to get club going at Abraham Lincoln High School

### ISSUE THREAD

**Issue by zachlatta** [hackedu/hackedu#84]
@MaxWofford can you reach out to Lincoln and see what needs to happen to get a club going there? The deliverable for this issue is creating issues for what to do next to get the club going.

/cc @jonleung @hellyeah

**Comment by jonleung** [comment]
@MaxWofford I know you are super busy. Do you have the bandwidth to do this? I'm happy to take this task if it is helpful. Thanks!

**Comment by MaxWofford** [comment]
Thank you @jonleung ! I'll assign you.

**Comment by jonleung** [comment]
:+1:

@MaxWofford can you PM me the teacher's email?

**Comment by MaxWofford** [comment]
Sent this info through IRC.

**Comment by jonleung** [comment]
:+1:

**Comment by jonleung** [comment]
Updates:

> Hi Jonathan,
>
> Glad to hear from you – the kids were asking. The kids voted to change gears a little bit with regards to the direction of the club. First, we are now the ROBOHACK Workshop (and I think you guys deserve some credit for this name). Second, we are expanding our focus beyond robots. Here are the areas we are interested in:
>
> Robots
> Art/music/kinetic sculpture (burning man style projects)
> Internet of Things / Arduino sensors / security
> Hacking ( including web and desktop programming)
> Other (solar panel research? Electric bikes? Who knows?)
>
> We will be looking for students and volunteers to get involved in each of these areas, and I obviously had you guys in mind for the hacking category (at the least).
>
> I’m available tomorrow before 9AM and after 2:10PM. Looking forward to hearing from you.

TLDR: Abraham Lincoln wants to do their own stuff. 

My thoughts are the we treat Abraham Lincoln like an "advanced" club that is trying different things than us. We can watch what they're doing and see what comes out of it but we won't be active in it. 

This means that we still need to find 1-2 schools in SF to start.

/cc @zachlatta @MaxWofford

SF to start our own clubs in...

**Comment by zachlatta** [comment]
Mmm, spending more time with Abraham Lincoln sounds like a distraction.

Let's start new clubs in SF elsewhere. @MaxWofford can you take point on
this? Or at least get the framework for getting this done set up before you
head off to BM?

On Thu, Aug 27, 2015 at 2:09 PM, Jonathan Leung <notifications@github.com>
wrote:

> Updates:
>
> Hi Jonathan,
>
> Glad to hear from you – the kids were asking. The kids voted to change
> gears a little bit with regards to the direction of the club. First, we are
> now the ROBOHACK Workshop (and I think you guys deserve some credit for
> this name). Second, we are expanding our focus beyond robots. Here are the
> areas we are interested in:
>
> Robots
> Art/music/kinetic sculpture (burning man style projects)
> Internet of Things / Arduino sensors / security
> Hacking ( including web and desktop programming)
> Other (solar panel research? Electric bikes? Who knows?)
>
> We will be looking for students and volunteers to get involved in each of
> these areas, and I obviously had you guys in mind for the hacking category
> (at the least).
>
> I’m available tomorrow before 9AM and after 2:10PM. Looking forward to
> hearing from you.
>
> TLDR: Abraham Lincoln wants to do their own stuff.
>
> My thoughts are the we treat Abraham Lincoln like an "advanced" club that
> is trying different things than us. We can watch what they're doing and see
> what comes out of it but we won't be active in it.
>
> This means that we still need to find 1-2 schools in SF to start.
>
> /cc @zachlatta <https://github.com/zachlatta> @MaxWofford
> <https://github.com/MaxWofford>
>
> SF to start our own clubs in...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackedu/hackedu/issues/84#issuecomment-135555660>.
>



-- 
Zach Latta
hackEDU | zach@hackedu.us

**Comment by zachlatta** [comment]
@hellyeah – would you be interested in being involved in starting new SF
clubs as well?

On Thu, Aug 27, 2015 at 2:26 PM, Zach Latta <zach@hackedu.us> wrote:

> Mmm, spending more time with Abraham Lincoln sounds like a distraction.
>
> Let's start new clubs in SF elsewhere. @MaxWofford can you take point on
> this? Or at least get the framework for getting this done set up before you
> head off to BM?
>
> On Thu, Aug 27, 2015 at 2:09 PM, Jonathan Leung <notifications@github.com>
> wrote:
>
>> Updates:
>>
>> Hi Jonathan,
>>
>> Glad to hear from you – the kids were asking. The kids voted to change
>> gears a little bit with regards to the direction of the club. First, we are
>> now the ROBOHACK Workshop (and I think you guys deserve some credit for
>> this name). Second, we are expanding our focus beyond robots. Here are the
>> areas we are interested in:
>>
>> Robots
>> Art/music/kinetic sculpture (burning man style projects)
>> Internet of Things / Arduino sensors / security
>> Hacking ( including web and desktop programming)
>> Other (solar panel research? Electric bikes? Who knows?)
>>
>> We will be looking for students and volunteers to get involved in each of
>> these areas, and I obviously had you guys in mind for the hacking category
>> (at the least).
>>
>> I’m available tomorrow before 9AM and after 2:10PM. Looking forward to
>> hearing from you.
>>
>> TLDR: Abraham Lincoln wants to do their own stuff.
>>
>> My thoughts are the we treat Abraham Lincoln like an "advanced" club that
>> is trying different things than us. We can watch what they're doing and see
>> what comes out of it but we won't be active in it.
>>
>> This means that we still need to find 1-2 schools in SF to start.
>>
>> /cc @zachlatta <https://github.com/zachlatta> @MaxWofford
>> <https://github.com/MaxWofford>
>>
>> SF to start our own clubs in...
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/hackedu/hackedu/issues/84#issuecomment-135555660>.
>>
>
>
>
> --
> Zach Latta
> hackEDU | zach@hackedu.us
>



-- 
Zach Latta
hackEDU | zach@hackedu.us

**Comment by jonleung** [comment]
Let's move this discussion to https://github.com/hackedu/hackedu/issues/136
